dipy/segment/bundles.py

```diff
@@ -615,7 +608,12 @@ def _prune_what_not_in_model(self, model_centroids,
         pruned_indices = [rtransf_cluster_map[i].indices
                           for i in np.where(mins != np.inf)[0]]
         pruned_indices = list(chain(*pruned_indices))
-        pruned_streamlines = transf_streamlines[np.array(pruned_indices)]
+        idx = np.array(pruned_indices)
+        if len(idx) == 0:
+            print(' You have removed all streamlines')
```

Comment:
`logging` instead of `print` ? or it should be merged before #1862

Reply:
If this is good, let's merge this one first, and then I can rebase #1862 and fix this up.

Reply:
sounds like a plan! merging